docs/HTTP_SPEC.md

```diff
@@ -0,0 +1,104 @@
+# Lassie: HTTP Specification
+
+![wip](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
+
+**Author(s)**:
+
+- [Hannah Howard](https://github.com/hannahhoward)
+- [Kyle Huntsman](https://github.com/kylehuntsman)
+- [Rod Vagg](https://github.com/rvagg)
+
+**Maintainer(s)**:
+
+- [Hannah Howard](https://github.com/hannahhoward)
+- [Kyle Huntsman](https://github.com/kylehuntsman)
+- [Rod Vagg](https://github.com/rvagg)
+
+* * *
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Specification](#specification)
+    - [`GET /ipfs/{cid}[?params]`](#get-ipfscidparams)
+
+
+## Introduction
+
+The Lassie HTTP Daemon is an HTTP interface for retrieving CAR files from the Filecoin network. It fetches content via CIDs over a number of possible protocols, choosing the best protocol for the job.
```

Comment:
This is a bit ambitious - "a number of possible protocols" and "choosing the best protocol for the job" are both aspirational